subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/AllDistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,7 @@ class AllDistributionIntegrationSpec extends DistributionIntegrationSpec {
 
     @Override
     int getMaxDistributionSizeBytes() {
-        return 193 * 1024 * 1024
+        return 201 * 1024 * 1024
```

Comment:
This makes sure that the distribution is not too large. I had to update it since I've been adding a lot of material (and this bytes) to the user manual.